frontend/src/components/LevelPage/TargetButton/TargetButton.scss

```diff
@@ -19,61 +46,59 @@
   }
 }
 
-.buttonPterodactyle {
-  all: unset;
+.t-rex {
+  width: 20vw;
+  position: absolute;
+  top: 45%;
+  left: 60%;
+  overflow: hidden;
+  &:hover {
+    filter: drop-shadow(3px 5px 2px rgb(0 0 0 / 0.2));
+    cursor: pointer;
+  }
+}
+.tRexPopUp {
+  width: 13vw;
+  margin-top: 1rem;
 }
 
-.ptero {
-  width: 10vw;
-  height: 10vh;
-  z-index: 1;
+.buttonPtetodactyle {
+  all: unset;
 }
 
-.pteroEgg {
-  width: 2vw;
-  height: 2vh;
+.pterodactylePopUp {
+  all: unset;
+  width: 15vw;
+  z-index: 0;
 }
 
-.popPtero {
-  position: absolute;
-  background-color: $primaryColor;
-  border: 2px solid $secondaryColor;
-  border-radius: 50px;
-  width: 30vw;
-  height: 30vh;
-  top: 20%;
-  left: 40%;
-  display: flex;
-  flex-wrap: wrap;
+.eggClass {
+  width: 2vw;
+  height: 4vh;
 }
 
-.name {
-  font-size: 3vw;
+.brain {
+  width: 4vw;
+  height: 8vh;
 }
 
 .info {
-  font-size: 0.5vw;
   flex: 1;
-  border-left: 0.125rem solid $secondaryColor;
-  border-right: 0.125rem solid $secondaryColor;
+  border-left: 2px solid $secondaryColor;
+  border-right: 2px solid $secondaryColor;
   height: 1.5vh;
-  margin-right: 1.2rem;
-  margin-left: 0.6rem;
-  padding-left: 0.3rem;
+  margin-right: 20px;
```

Comment:
Responsive values for margins and padding please

Reply:
Modified